assets/scss/modules/_navigation.scss

```diff
@@ -152,6 +152,12 @@ a.header__menu__proposition-name {
   }
 }
 
+.tab-content {
+  .first {
```

Comment:
can this BEM name have more context? only really has context here when it is shown inside `.tab-content` could be `.tab-content__first`